docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md

```diff
@@ -14,6 +14,24 @@ title: Test backing image
 9. Create and attach a volume with the backing image. Verify the data content is correct.
 10. Do cleanup.
 
+### Upload via Rancher URI
+<sup>Related issue: [3129](https://github.com/longhorn/longhorn/issues/3129)</sup>
+1. Create or import cluster into Rancher
```

Comment:
Can you mention the Rancher series here, is it v2.5.x or v2.6.x?

Reply:
Thanks for the reminder. Adding in next commit.